UPDATING.md

```diff
@@ -25,31 +25,37 @@ assists people when migrating to a new version.
 ## Next
 
 ### Breaking Changes
+
```

Comment:
looks like we got some autogenerated changes here. maybe we should move those into another pr (if we think they're desirable?)

Reply:
this pr is fine. I checked and it looks the same as the old with my new changes. Not sure why it did all this in the diff.